lib/dataContract/stateTransition/DataContractCreateTransition.js

```diff
@@ -59,48 +65,71 @@ class DataContractCreateTransition extends AbstractStateTransitionIdentitySigned
    * Get state transition as plain object
    *
    * @param {Object} [options]
-   * @param {boolean} [options.skipSignature]
-   *
-   * @return {Object}
+   * @param {boolean} [options.skipSignature=false]
+   * @param {boolean} [options.encodedBuffer=false]
+   * @return {RawDataContractCreateTransition}
    */
   toObject(options = {}) {
-    return {
+    Object.assign(
+      options,
+      {
+        encodedBuffer: false,
+        ...options,
+      },
+    );
+
+    const rawStateTransition = {
       ...super.toObject(options),
-      dataContract: this.getDataContract().toJSON(),
-      entropy: this.entropy,
+      dataContract: this.getDataContract().toObject(),
+      entropy: this.getEntropy(),
     };
+
+    if (!options.encodedBuffer) {
+      rawStateTransition.entropy = this.getEntropy().toBuffer();
+    }
+
+    return rawStateTransition;
   }
 
   /**
-   * Get owner ID
-   * @return {string}
+   * Get state transition as JSON
+   *
+   * @return {JsonDataContractCreateTransition}
    */
-  getOwnerId() {
-    return this.getDataContract().getOwnerId();
+  toJSON() {
+    return {
+      ...super.toJSON(),
+      entropy: this.getEntropy().toString(),
+      dataContract: this.getDataContract().toJSON(),
+    };
   }
 
   /**
-   * Create state transition from JSON
-   *
-   * @param {RawDataContractCreateTransition} rawStateTransition
-   *
-   * @return {DataContractCreateTransition}
+   * Get owner ID
+   * @return {EncodedBuffer}
    */
-  static fromJSON(rawStateTransition) {
-    return new DataContractCreateTransition(
-      AbstractStateTransitionIdentitySigned.translateJsonToObject(rawStateTransition),
-    );
+  getOwnerId() {
+    return this.getDataContract().getOwnerId();
   }
 }
 
 /**
- * @typedef {Object} RawDataContractCreateTransition
- * @property {number} protocolVersion
- * @property {number} type
+ * @typedef {RawStateTransitionIdentitySigned & Object} RawDataContractCreateTransition
  * @property {RawDataContract} dataContract
- * @property {number|null} signaturePublicKeyId
- * @property {string|null} signature
- * @property {string|null} entropy
+ * @property {Buffer} entropy
+ */
+
+/**
+ * @typedef {JsonStateTransitionIdentitySigned & Object} JsonDataContractCreateTransition
```

Comment:
What does it mean? JsonStateTransitionIdentitySigned & Object

Reply:
https://github.com/jsdoc/jsdoc/issues/1199#issuecomment-457227207